app/CMakeLists.txt

```diff
@@ -2,6 +2,8 @@ cmake_minimum_required(VERSION 3.13.1)
 
 find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 
+zephyr_compile_definitions(CONFIG_BUILD_TYPE="${EAST_BUILD_TYPE}")
```

Comment:
@MarkoSagadin Please also think about if we are ok injecting the east build type like this.
I added this here to be able to print all of the same information that `version-info` did. But it is not very important. 

Any thoughts?

Reply:
I would remove this.